docs/routing/index.md

```diff
@@ -1,10 +1,11 @@
 # Routing
 
-Now we have all the basic concepts of vuejs, its components & lifecycles and how to communicate between them. So far so good, but most of modern UI applications are not simply static sites right?. How about navigating between pages and dynamically load different components based on that navigation? Vuejs provides a powerfull plugin library for those use cases where your application will dynamically load different components based on some kind of routing. That's all we know as Single Page Applications.
+Now we have all the basic concepts of Vue.js, its components and lifecycles and how to communicate between them. So far so good, but most of modern UI applications are not simply static sites right?.
```

Comment:
I suggest to change the first line of the first item here like this:

"Now we know the basis of Vue.js, we know how to create components, use lifecycle features and communicate multiple components. Quite a few useful things, but for most modern UI applications are usually more dynamic than this."